src/domains/sign/integration.tf

```diff
@@ -0,0 +1,121 @@
+module "event_hub" {
+  source                   = "git::https://github.com/pagopa/terraform-azurerm-v3.git//eventhub?ref=v4.1.7"
```

Comment:
I'd add a description ("integration" is not easy to grasp at first glance)

Reply:
As above, this terraform file is aimed at collecting all the resources required to integrate our domain with external domains, products or platforms.

One possible way could be via a comment, but why commenting this and not the others resource groups?

Reply:
Done